cloud-deployment-guide/cloud-troubleshooting.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Cloud Troubleshooting
+description_short: Troubleshooting various cloud errors
+description: Causes and solutions to common issues when using Tiny Cloud
+keywords: tinymce cloud script textarea apiKey troubleshooting banners domain referer
+---
+
+When attemting to load the editor, many possible problems can occur. These are mostly related to a incorrect combination of 
+your `apiKey` and the page `referer`. When {{site.cloudname}} detects a problem, it will show an editor
+notification containing information about the problem. You can find explanations for the causes and solutions to 
+the {{site.cloudname}} notifications here.
+
+> NOTE: The wording of these notifications might differ slightly from what {{site.cloudname}} is showing.
+
+## Editor Notifications
+
+### "This domain is not registered with Tiny Cloud. Please see the quick start guide or create an account."
+
+#### Cause
+
+This notification is **only shown** when you have either:
+
+1. not provided an `apiKey` in the script tag.
+1. used `no-api-key` as your apiKey
+
+#### Solution
+
+If you have already signed up for {{site.cloudname}}, then you need to ensure that your {{site.cloudname}} script source uses that apiKey. It should **not** include `no-api-key`.
+
+If you have signed up for {{site.cloudname}}, you will need to signup to receive an apiKey.
+
+### "The API key you have entered is invalid. Please review your API key."
+
+#### Cause
+
+This notification is shown when you have provided an `apiKey`, but that `apiKey` has the wrong structure. Your `apiKey` must
```

Comment:
- [ ] I'm pretty sure this is wrong actually. I should double-check it. I think it will give this error if the apiKey can't be found as well.

Reply:
Yeah, it's wrong. Need to fix.

Reply:
```
### Cause

This notification is shown when either:

- The API key provided cannot be found on the {{site.cloudname}} server.
- The provided `apiKey` has the wrong structure.
```
?

Reply:
I'm not `100%` sure on the structure, actually .. but the "cannot be found" will true for an invalid instructure anyway, so maybe just say "cannot be found"